tests/testthat/test-valid_years.R

```diff
@@ -0,0 +1,18 @@
+
+test_that("valid_years returns expected output type", {
+  tmp <- valid_years('../testdata/app_data/20210401/')
```

Comment:
I modified the path here, because the unit tests were using your local path to the `fake-pip-data` repo